images/linux/Ubuntu2004-Readme.md

```diff
@@ -99,32 +99,32 @@
 - Skopeo 1.5.0 (apt source repository: https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable)
 - Sphinx Open Source Search Server 2.2.11
 - SVN 1.13.0
-- Terraform 1.2.7
+- Terraform 1.2.8
 - yamllint 1.27.1
 - yq 4.27.2
 - zstd 1.5.2 (homebrew)
 
 ### CLI Tools
-- Alibaba Cloud CLI 3.0.124
-- AWS CLI 2.7.25
+- Alibaba Cloud CLI 3.0.125
+- AWS CLI 2.7.27
 - AWS CLI Session manager plugin 1.2.339.0
 - AWS SAM CLI 1.55.0
 - Azure CLI (azure-cli) 2.39.0 (installation method: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt)
 - Azure CLI (azure-devops) 0.25.0
-- GitHub CLI 2.14.4
+- GitHub CLI 2.14.7
 - Google Cloud SDK 369.0.0 (apt source repository: https://packages.cloud.google.com/apt)
 - Hub CLI 2.14.2
-- Netlify CLI 11.3.0
-- OpenShift CLI 4.11.0
-- ORAS CLI 0.13.0
-- Vercel CLI 28.1.0
+- Netlify CLI 11.5.0
+- OpenShift CLI 4.11.1
+- ORAS CLI 0.14.0
+- Vercel CLI 28.2.0
 
 ### Java
 | Version             | Vendor          | Environment Variable |
 | ------------------- | --------------- | -------------------- |
 | 8.0.345+1           | Eclipse Temurin | JAVA_HOME_8_X64      |
-| 11.0.16+8 (default) | Eclipse Temurin | JAVA_HOME_11_X64     |
-| 17.0.4+8            | Eclipse Temurin | JAVA_HOME_17_X64     |
+| 11.0.16+1 (default) | Eclipse Temurin | JAVA_HOME_11_X64     |
+| 17.0.4+1            | Eclipse Temurin | JAVA_HOME_17_X64     |
```

Comment:
@mikhailkoliada is it downgrading?

Reply:
It's not downgrading, the previous version was 11.0.16.0.0+8 vs 11.0.16.1.0+1 now, but we should preserve this version syntax to not breaking toolset for java.

Reply:
maybe we should rework our software generator then to output the correct version?

Reply:
I wonder what happened who starts using version notation from software generator syntax in setup-java actions?

Reply:
Filed an issue https://github.com/actions/runner-images-internal/issues/4242